.github/workflows/package.yml

```diff
@@ -77,6 +77,17 @@ jobs:
       fail-fast: false
       matrix:
         package-id:
+          - postgresql-16-pgdg-pgroonga-debian-bookworm
+          - postgresql-16-pgdg-pgroonga-debian-bookworm-arm64
+          - postgresql-16-pgdg-pgroonga-debian-bullseye
+          - postgresql-16-pgdg-pgroonga-debian-bullseye-arm64
```

Comment:
I don't want to add new packages for oldstable...
How about removing them?

Reply:
I see. I've removed them.